addresspoints/src/main/scala/grasshopper/addresspoints/model/Status.scala

```diff
@@ -0,0 +1,3 @@
+package grasshopper.addresspoints.model
+
+case class Status(status: String, service: String, time: String, host: String)
```

Comment:
I added an `upSince` attribute to grasshopper-parser's `/status` [response](https://github.com/cfpb/grasshopper-parser#response).  Thoughts on adding that here as well?

Also, looks like I need to add a `service` attribute there as well so they line up.  Just added https://github.com/cfpb/grasshopper-parser/issues/15 for that.